fetch.js

```diff
@@ -294,7 +299,9 @@
   }
 
   Request.prototype.clone = function() {
-    return new Request(this)
+    var newRequest = new Request(this, { body: 'clone' })
```

Comment:
How about passing `this._bodyInit` to `body:` parameter instead of `'clone'` which will be discarded anyway?